MekHQ/src/mekhq/campaign/io/CampaignXmlParser.java

```diff
@@ -581,11 +583,59 @@ public Campaign parse() throws CampaignXmlParseException, NullEntityException {
                         }
                     });
 
+
+
+        if (MHQConstants.VERSION.isHigherThan(version)) {
+            triggerActiveContractWarning(retVal);
+        }
```

Comment:
I think I'd prefer a trigger/warning in somewhere like MekHQ or in a GUI, not in the XML Parser. static checks like returning the information if it is in the correct version or not should be done in a place specific to the campaign + application.

I'm not hard on this, could go either way, just don't like calling a GUI think inside an XML parser. To me this kind of thing has to be handled by the application, otherwise we could have something like a GUI less application trying to run in the CMD and throwing errors when loading the XML because it requires a frame.